randomizer.py

```diff
@@ -998,6 +1036,26 @@ def potionCards(self):
 Traits = Plunder.traits
 LandscapeCards = Events | Landmarks | Projects | Ways | Traits
 
+# Define action cards
+ActionCards = (
```

Comment:
The "card type" variables above are not suffixed with "Cards," the exception being the landscape cards, and of course, "landscape" is not a card type. This variable should just be `Actions`